src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.ImportWorkloads.targets

```diff
@@ -35,4 +35,32 @@ Copyright (c) .NET Foundation. All rights reserved.
                           NETCoreSdkVersion="$(NETCoreSdkVersion)"
                           GenerateErrorsForMissingWorkloads="true"/>
   </Target>
+
+  <Target Name="_SetSkipResolvePackageAssets" >
+    <PropertyGroup>
+      <SkipResolvePackageAssets>true</SkipResolvePackageAssets>
+    </PropertyGroup>
+  </Target>
```

Comment:
Why have this mutate state within the build rather than pass it in?

Reply:
just set global state? If so the target will not be able to function by itself and there is no good way to tell you need to set SkipResolvePackageAssets.

Reply:
I'm mostly worried about calling this target somehow during a real build and it being a nightmare to debug. It is "private" though . . .

Reply:
Set globalproperty instead